src/psyclone/tests/psyir/tools/definition_use_chains_backward_dependence_test.py

```diff
@@ -228,6 +229,38 @@ def test_definition_use_chain_find_backward_accesses_ifelse_example(
     assert reaches[1] is not a_3
 
 
+def test_definition_use_chain_find_backward_accesses_psy_data_node_example(
+    fortran_reader,
+):
+    """Functionality test for the find_backward_accesses routine. This
+    tests the behaviour when the behavious is the expected when the analysed
```

Comment:
s/when the behavious is the/is as/

Reply:
Please add some text to say what the expected behaviour is.

Reply:
Done